README.md

```
@@ -1,2 +1,198 @@
+
+전남대학교 사물함 신청 서비스는 학생들이 보다 편리하게 사물함을 신청하고 위원회에서 하나의 플랫폼에서 효율적으로 사물함을 통합 관리할 수 있는 서비스입니다. 
+
+
+## 주요 특징
+
+### 사물함 신청하기 만들기
+
+이벤트 기본 정보를 설정하고 사물함 신청 참여 학과를 설정함으로써 사물함 신청하기를 만들 수 있습니다.
+
+<p align="center">
+  <img src="https://velog.velcdn.com/images/dubu1001/post/3dc39bbe-7bac-4c6a-81ce-c4c873cb73fc/image.png" width="500px">
```

markdownlint-cli2 (0.17.2)

Images should have alternate text (alt text)
null

(MD045, no-alt-text)